Makefile

```diff
@@ -111,7 +111,7 @@ docker-genassets:
 	\
 	rm -f bin/tarfs bin/rsrc && \
 	go install github.com/getlantern/tarfs/tarfs && \
-	echo "// +build prod" > $$DEST && \
+	echo "// +build !stub" > $$DEST && \
```

Comment:
Instead of trying to fix prod I decided to add a different build tag that is never used.